### PR TITLE
zee: depends on boost and git

### PR DESCRIPTION
**Pull request by tristan0x**

### CODE REVIEW

File: var/spack/repos/builtin/packages/zee/package.py [hunk 1]
```diff
@@ -30,8 +30,10 @@ class Zee(CMakePackage):
     variant('codechecks', default=False,
             description='Perform additional code checks like ' +
                         'formatting or static analysis')
+    depends_on('boost')
```
Comment (pramodk) [inline var/spack/repos/builtin/packages/zee/package.py:33]: Just to understand: are we using boost explicitly in the zee core or just got tests?

File: var/spack/repos/builtin/packages/zee/package.py [hunk 2]
```diff
@@ -30,8 +30,10 @@ class Zee(CMakePackage):
...
     depends_on('cmake@3:', type='build')
     depends_on('pkg-config', type='build')
+    depends_on('git', type='build', when='+codechecks')
```
Comment (pramodk) [inline var/spack/repos/builtin/packages/zee/package.py:36]: git is prerequisite with spack so I would remove it.

**Review by pramodk — CHANGES_REQUESTED** [review]

File: var/spack/repos/builtin/packages/zee/package.py (hunk 1 above)
Comment (tristan0x) [reply var/spack/repos/builtin/packages/zee/package.py:33]: I am using boost timer explicitely

File: var/spack/repos/builtin/packages/zee/package.py (hunk 2 above)
Comment (tristan0x) [reply var/spack/repos/builtin/packages/zee/package.py:36]: I need to execute command `git merge-base --fork-point` which is not supported by `/usr/bin/git` on BB5

Comment (matz-e) [reply var/spack/repos/builtin/packages/zee/package.py:36]: Ancient RH git… :/

**Review by matz-e — APPROVED** [review]

File: var/spack/repos/builtin/packages/zee/package.py (hunk 1 above)
Comment (pramodk) [reply var/spack/repos/builtin/packages/zee/package.py:33]: Ok (that's seems bit heavy dependency for timer though!)

**Review by pramodk — APPROVED** [review]

File: var/spack/repos/builtin/packages/zee/package.py (hunk 1 above)
Comment (tristan0x) [reply var/spack/repos/builtin/packages/zee/package.py:33]: > Ok (that's seems bit heavy dependency for timer though!)

I will use `std::chrono` library instead, but I like to have boost available if needed.

**Pull request merged**